packages/manager/apps/telecom/src/app/telecom/pack/xdsl/access/translations/Messages_fr_FR.json

```diff
@@ -107,7 +107,8 @@
   "xdsl_access_amount_upload": "Volume émis dans le mois",
   "xdsl_details_action": "Actions",
   "xdsl_details_transfert": "Modifier mon offre",
-  "xdsl_modem_comfort_exchange_subtitle": "Échange de modem",
+  "xdsl_modem_comfort_exchange_subtitle": "Échange de modem de confort",
   "xdsl_modem_exchange": "Échanger mon modem",
-  "xdsl_modem_exchange_title": "Échanger"
+  "xdsl_modem_exchange_title": "Échanger",
+  "xdsl_modem_comfort_exchange_description": "Vous avez la possibilité de changer votre modem existant vers un modem Zyxel VMG8824.<br>Le coût de l'échange est de 70€ HT (soit 84€TTC).<br><br>Le nouveau modem sera envoyé automatiquement à l'adresse de raccordement."
```

Comment:
Please remove the `br`

Can the model type be retrieved from the API ?

Can the price be retrieved from the API ?

Reply:
It will but later. For the moment, we need to update this label like that to inform customers that this service is not free and which they will pay.